lib/active_model/serializer/association.rb

```diff
@@ -9,10 +9,58 @@ class Serializer
     # @example
     #  Association.new(:comments, CommentSummarySerializer)
     #
-    Association = Struct.new(:name, :serializer, :options, :links, :meta) do
+    Association = Struct.new(:name, :serializer, :options) do
```

Comment:
this should probably just subclass  `ActiveModel::Serializer::Field` like everything else. see https://github.com/rails-api/active_model_serializers/pull/1857/files#r73813576